portfolio.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" type="text/css" href="normalize.css">
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<link href="https://fonts.googleapis.com/css?family=Didact+Gothic&display=swap" rel="stylesheet">
+		<title>Isabella Aston's Portfolio</title>
+	</head>
+	<body>
+		<div class="open" id="top">
+			<nav>
+				<div class="menu-right">
+					<a href="#top">Home</a>
+					<a href="#about">About Me</a>
+					<a href="#portfolio">Portfolio</a>
+					<a href="#contact">Contact Me</a>
+			</nav>
+			</div>
+			<div class="socials">
+					<h1>Isabella Aston</h1>
+					<div class="social-links">
+						<h1>Full Stack Developer</h1>
+						<a href="https://github.com/isabellaaston">
+						<img class="logos" src="github.png" alt="GitHub"></a>
+						<a href="https://www.linkedin.com/in/isabella-aston/">
+						<img class="logos" src="linkedin.png" alt="LinkedIn"></a>
+						<a href="https://www.instagram.com/this_izz/">
+						<img class="logos" src="instagram.png" alt="Instagram"></a>
```

Comment:
Might be a good idea to make these links open in a new tab so users don't get taken away from your website.